### DISCUSSION TITLE
Unable to farm 24 hours after joining the mining pool

### DISCUSSION THREAD

**Discussion by qq597353** [Chia-Network/chia-blockchain#7449] [Support]
![chia pool](https://user-images.githubusercontent.com/26955326/125469583-4278cad0-79c6-415a-9060-765c7335f0b4.jpg)
Unable to farm 24 hours after joining the mining pool

**Comment by cimrhanzel** [comment]
Intel MacOs version ? 
Are you getting this in your logs  
"2021-07-13T10:28:39.553 farmer chia.farmer.farmer         : ERROR    Exception in GET /pool_info https://Pool_Name_here, Cannot connect to host na1.pool.space:443 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')]"

**Reply by qq597353** [reply]
M1

**Reply by qq597353** [reply]
unsupported operand type(s) for /: 'int' and 'NoneType'

**Reply by cimrhanzel** [reply]
i get that if the pool requires a sign in.  They did not test the MacOs versions before release.

**Reply by qq597353** [reply]
What should I do now?

**Comment by STATION-I** [comment]
Same issues but on Ubuntu. STAI with same plots on same system is working fine. Have you found a solution?

**Comment by TianciD** [comment]
Same issues

**Comment by loppefaaret** [comment]
if any one of you could locate the `debug.log` on your farming machine, it is by default under your users home directory, in `~/.chia/mainnet/log/debug.log` and attach it to this issue please
also what macOS version you are using
